numpy/__init__.pyi

```diff
@@ -202,6 +202,7 @@ from numpy import (
     rec as rec,
     testing as testing,
     version as version,
+    CopyMode as CopyMode
```

Comment:
This isn't going to work I'm afraid, as there is no `CopyMode` module and neither can you import the object from the very same module that you're trying it import it to (bit of a circular reasoning...).

Instead, you'll have to define annotations within the stub file, preferably somewhere near the bottom.
``` python
class CopyMode(enum.Enum):  # Don't forget to import `enum`
    ALWAYS: L[1]
    IF_NEEDED: L[0]
    NEVER: L[2]
```